app/models/bus_stop.rb

```diff
@@ -67,7 +67,8 @@ def route_list
   def self.to_csv(limited_attributes: false)
     attrs = if limited_attributes
               LIMITED_ATTRS
-            else Hash[columns.map{|c| [c.name, c.name.humanize] }]
+            else
+              Hash[columns.map{ |c| [c.name, c.name.humanize] }].merge(route_list: 'Routes')
```

Comment:
Simpler I think to merge all of `LIMITED_ATTRS`.

Reply:
I forgot this comment oh no

Reply:
But isn't limited_attributes false? Why would I merge `LIMITED_ATTRS` then?

Reply:
Oh, actually. I see now.

Reply:
But then I have columns of `Hastus ID`, `Stop Name`, and `Last Updated` seeing as `Hastus`, `Name`, and`Updated at` aren't equal to `Hastus ID`, `Stop Name` and `Updated at` respectively.

Reply:
I don't think what you're describing is how `Hash#merge` works. Could you give an example of what you're saying you don't want, but are getting?

Reply:
Yes, of course

Reply:
First, the change that you're talking about... would it look like
`Hash[columns.map{ |c| [c.name, c.name.humanize] }].merge LIMITED_ATTRS`